### PR TITLE
adds admin permissions

### PR DESCRIPTION
**Pull request by scmountain**
85-0-2

### CODE REVIEW

File: spec/features/admin_permission_spec.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    it "create user" do
+      visit signup_path
+
+      expect(current_path).to_not eq(signup_path)
+      expect(current_path).to eq(root_path)
+    end
+    it "view user index" do
+      visit users_path
+
+      expect(current_path).to eq(users_path)
```
Comment (CjMoore) [inline spec/features/admin_permission_spec.rb]: Tests look really solid.

Comment (scmountain) [inline spec/features/admin_permission_spec.rb]: built off of yours to help maintain coverage and quickly get these permissions into the teams hands...

Comment (CjMoore) [inline spec/features/admin_permission_spec.rb]: gracias 🕺

Comment (edilenedacruz) [inline spec/features/admin_permission_spec.rb]: @CjMoore rocks!

**Pull request merged**